Book-A-Desk.Api/BookingsHttpHandler.fs

```diff
@@ -65,6 +67,10 @@ module BookingsHttpHandler =
                             Date = booking.Date
                             User = { Email = booking.User.Email }
                         }
+                    let! sent = notifySuccess booking
+                    match sent with
```

Comment:
if notifySuccess returns a unit, then it is guaranteed to return (). You don't need to match. You can just do:
``` fsharp
do! notifySuccess booking
printfn "I've sent a mail message!"
```

Reply:
I've changed the behaviour to get if process has failed and manage the response.